about.html

```diff
@@ -0,0 +1,134 @@
+    </div>
+
+    <h4 class="text-center fs-3 mt-5 " style="color: rgb(0, 0, 0);">See the past Africa Day 2023 concert in Ethiopia</h4>
+    <hr class='mb-5' style="color:rgb(255, 51, 0,1) ;border: 2px solid rgb(255, 51, 0,1);width: 20%; margin-left: 40%;">
+    <p class="text-center twocassions">look at the past two Africa Day 2023 concert which took place in Ethiopia </p>
+
+    <div class="imgwrapper justify-content-center">
+        <div class="singleimg">
+            <div class="imgabout p-0" style="width: 100%;">
+                <img src="./assets/images/past-event-images/imager.jpg" alt="about image 1" width="500" height="400">
+
+                <div class="redmask">
+                    <div>
+                        <h6 class='ps-4' style="color: rgb(255, 255, 255);font-weight: lighter;">
+                            3rd May 2023
+                        </h6>
+                        <p style="color: white;font-weight: bolder;font-size: 28px;">Helton Hotel</p>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="singleimg">
+            <div class="imgabout p-0" style="width: 100%;">
+                <img src="./assets/images/past-event-images/musium.jpg" alt="about image 2" width="500" height="400">
+
+                <div class="redmask">
+                    <div>
+                        <h6 class='ps-5' style="color: white;">
+                            2nd June 2023
+                        </h6>
+                        <p style="color: white;font-weight: bolder;font-size: 28px;">Sheraten Hotel</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <section id="sponsor" class="partners text-center dark-bg-color text-secondary-white"
+                    style="background-color:#414246;">
+        <h4 class="text-center fs-2 mt-5" style="color: #ffffff;">Partners</h4>
+        <hr class='mb-4' style="color:#a04038 ;border: 2px solid #a04038;width: 10%; margin-left: 45%;">
+
+        <div class="d-flex justify-content-center group2">
+                <div class="partners-logo logos-img">
+                  <a href="https://www.mozilla.org/en-US/firefox/"><img src="./assets/images/partners-images/mozilla 1.svg" alt="mozilla"></a>
+                  <a href="https://www.google.com/"><img src="./assets/images/partners-images/googlekorea 1.svg" alt="google"></a>
+                  <a href="https://www.naver.com/"><img src="./assets/images/partners-images/naver 1.svg" alt="naver"></a>
+                </div>
+                <div class="partners-logo logos-img">
+                  <a href="http://www.kakaocorp.com/"><img src="./assets/images/partners-images/daumkakaologo 1.svg" alt="daumkakao"></a>
+                  <a href="https://www.airbnb.com/"><img src="./assets/images/partners-images/airbnb 1.svg" alt="iairbnb"></a>
+                </div>
```

Comment:
- [ ] Please make sure that in the mobile version of your index page partner section, the logos are in two lines just like in the design guideline.to meet exactly the expected design
- your design to be fixed 👇 
<img width="370" alt="Screen Shot 2023-06-30 at 23 12 11" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/60146030/b0a3c55b-4685-4c9f-a19e-65f1b39c1f5a">